src/include/duckdb/common/types/vector.hpp

```diff
@@ -304,6 +304,8 @@ struct FlatVector {
 		return !vector.validity.RowIsValid(idx);
 	}
 	DUCKDB_API static const SelectionVector *IncrementalSelectionVector();
+	static vector<idx_t> Search(Vector &keys, idx_t count, const Value &key, list_entry_t &entry);
```

Comment:
Perhaps these can be moved to a dedicated `MapVector` class instead?

Reply:
Done!